app/models/user.rb

```diff
@@ -1,26 +1,35 @@
 class User < ActiveRecord::Base
   include ModelsModule
-  attr_accessible :name, :okta_name, :email, :admin, :associate_consultant_attributes
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  attr_accessible \
+      :name,
+      :okta_name,
+      :email,
+      :admin,
+      :associate_consultant_attributes
 
-  has_many :coachees, :class_name => "AssociateConsultant", :foreign_key => "coach_id"
-  has_many :additional_emails, class_name: "AdditionalEmail",
-    foreign_key: "user_id", dependent: :destroy
-  has_one :associate_consultant, :dependent => :destroy
   has_many :feedbacks
+  has_many :coachees, class_name:'AssociateConsultant', foreign_key: 'coach_id'
+  has_many :additional_emails, class_name: 'AdditionalEmail',
+    foreign_key: 'user_id', dependent: :destroy
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.